soql-server-pg/src/main/scala/com/socrata/pg/server/analyzer2/ProcessQuery.scala

```diff
@@ -252,32 +255,38 @@ object ProcessQuery {
         }
       }
 
-    ETag(etag) ~> LastModified(lastModified) ~> Write("application/json; content-type=utf-8") { writer =>
-      writer.write("[{")
-      for(debugFields <- debugFields) {
-        writer.write("\"debug\":%s\n ,".format(JsonUtil.renderJson(debugFields, pretty = false)))
-      }
-      writer.write("\"fingerprint\":%s\n ,".format(JString(Base64.getUrlEncoder.withoutPadding.encodeToString(etag.asBytes))))
-      writer.write("\"last_modified\":%s\n ,".format(JString(CJSONWriter.dateTimeFormat.print(lastModified))))
-      writer.write("\"locale\":%s\n ,".format(JString(locale)))
-      writer.write("\"schema\":")
+    ETag(etag) ~> LastModified(lastModified) ~> ContentType("application/x-socrata-gzipped-cjson") ~> Stream { rawOutputStream =>
```

Comment:
and this is the signal throughout the stack that transparent decompression/recompression should not happen.